astronomer/providers/google/cloud/sensors/gcs.py

```diff
@@ -12,38 +11,17 @@
     GCSUploadSessionCompleteSensor,
 )
 
-from astronomer.providers.google.cloud.triggers.gcs import (
-    GCSBlobTrigger,
-    GCSCheckBlobUpdateTimeTrigger,
-    GCSPrefixBlobTrigger,
-    GCSUploadSessionTrigger,
-)
-from astronomer.providers.utils.sensor_util import poke, raise_error_or_skip_exception
-from astronomer.providers.utils.typing_compat import Context
-
 
 class GCSObjectExistenceSensorAsync(GCSObjectExistenceSensor):
     """
-    Checks for the existence of a file in Google Cloud Storage.
-
-    :param bucket: The Google Cloud Storage bucket where the object is.
-    :param object: The name of the object to check in the Google cloud storage bucket.
-    :param google_cloud_conn_id: The connection ID to use when connecting to Google Cloud Storage.
-    :param delegate_to: (Removed in apache-airflow-providers-google release 10.0.0, use impersonation_chain instead)
-        The account to impersonate using domain-wide delegation of authority, if any. For this to work, the service
-        account making the request must have domain-wide delegation enabled.
-    :param impersonation_chain: Optional service account to impersonate using short-term
-        credentials, or chained list of accounts required to get the access_token
-        of the last account in the list, which will be impersonated in the request.
-        If set as a string, the account must grant the originating account
-        the Service Account Token Creator IAM role.
-        If set as a sequence, the identities from the list must grant
-        Service Account Token Creator IAM role to the directly preceding identity, with first
-        account from the list granting this role to the originating account (templated).
+    This class is deprecated.
+    Please use :class: `~airflow.providers.google.cloud.sensors.gcs.GCSObjectExistenceSensor`
+    and set `deferrable` param to `True` instead.
     """
 
     def __init__(
         self,
+        *args: Any,
         polling_interval: float = 5.0,
```

Comment:
can we check if we need to pass this in the kwargs["poke_interval"] = polling interval instead of self.poke_interval=polling_interval?

Meaning to check if self.poke_interval will be used or if we need to set it in kwargs before calling super.init

Reply:
yes, we'll need to do that. just updated it